src/storage/src/hummock/sstable/multi_builder.rs

```diff
@@ -195,15 +188,7 @@ where
         let mut last_range_tombstone_epoch = HummockEpoch::MAX;
         if let Some(builder) = self.current_builder.as_mut() {
             if is_new_user_key {
-                if switch_builder {
-                    need_seal_current = true;
-                } else if builder.reach_capacity() {
-                    if !self.is_target_level_l0_or_lbase || builder.reach_max_sst_size() {
-                        need_seal_current = true;
-                    } else {
-                        need_seal_current = self.is_target_level_l0_or_lbase && vnode_changed;
-                    }
-                }
+                need_seal_current = switch_builder || builder.reach_capacity();
```

Comment:
we can also remove `reach_max_sst_size` and `is_target_level_l0_or_lbase` ?

Reply:
fixed